### ISSUE TITLE
Wifi settings

### ISSUE THREAD

**Issue by DirkTas67** [sidoh/esp8266_milight_hub#858]
I installed the 1.13.0 version recently and I can't find the option to change wifi settings (wifi network, wifi password).
Is it somewhere and I missed it?

**Comment by sidoh** [comment]
There actually never was a setting to change wifi network. It's possible to add, but a little tricky. As a workaround, you can create a backup, wipe flash, go through wifi setup and restore the backup.

**Comment by DirkTas67** [comment]
Ok, will do that, I never needed this option before so I probably didn't notice, but now I want to change the wifi network.
Thank you.

**Comment by DirkTas67** [comment]
Updated my Milight hub today, so took a backup of the current version (1.13.0), grabbed the device, and tried to flash the device in chrome with web.esphome.io, it worked, first everything was erased and then flashed, then connected to the device directly, entered my new wifi credentials, logged in via the network and was presented suddenly with a user and password request, tried my old settings but nothing worked. Also tried the beta2, tried with esptool (with erasing first) but all gave the same result.
In the end I flashed 1.11.2 restored a older backup I had, that worked and then I updated again to 1.13.1 beta2.
Everything is fine now.

Regards,
Dirk
